### PR TITLE
Change default activation to LeakyReLU in Conv layers

### PR DESCRIPTION
**Pull request by Logesh1502**
## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

### 🌟 Summary
Switches the default activation function in core convolutional modules from SiLU to LeakyReLU.

### 📊 Key Changes
- Updates `Conv.default_act` to `nn.LeakyReLU(negative_slope=0.1, inplace=True)`.
- Applies the same default activation change to `ConvTranspose` and `RepConv`.
- Does not alter layer interfaces or explicitly configured activation behavior.

### 🎯 Purpose & Impact
- Provides LeakyReLU as the default activation across these convolution layers.
- May affect model training dynamics, convergence, accuracy, and compatibility with existing pretrained weights.
- Requires validation across supported tasks and model configurations because this is a behavior-changing default.

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @Logesh1502, thank you for submitting a `ultralytics/ultralytics` 🚀 PR! This automated message helps with the initial review, and an engineer will assist as needed. To ensure a seamless integration of your work, please review the following checklist:

- ✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and link to any [relevant issues](https://github.com/ultralytics/ultralytics/issues). Ensure your commit messages are clear, concise, and adhere to the project's conventions.
- ✅ **Synchronize with Source**: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI) checks are passing. If any checks fail, please address the issues.
- ✅ **Update Documentation**: Update the relevant [documentation](https://docs.ultralytics.com/) for any new or modified features.
- ✅ **Add Tests**: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
- ✅ **Sign the CLA**: Please ensure you have signed our [Contributor License Agreement](https://docs.ultralytics.com/help/CLA) if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
- ✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for your bug fix or feature addition. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_ — Bruce Lee

For more guidance, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing). Don't hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🚀

File: ultralytics/nn/modules/conv.py [hunk 1]
```diff
@@ -226,7 +226,7 @@ class ConvTranspose(nn.Module):
         default_act (nn.Module): Default activation function (SiLU).
     """
 
-    default_act = nn.SiLU()  # default activation
+    default_act = nn.LeakyReLU(negative_slope=0.1, inplace=True)  # default activation
```
Comment (UltralyticsAssistant) [inline ultralytics/nn/modules/conv.py:229]: 💡 **MEDIUM**: This separate class default is not updated by a YAML `activation:` setting: `parse_model()` only assigns `Conv.default_act`. As a result, a custom model that explicitly requests (for example) `nn.SiLU()` will still use LeakyReLU for `ConvTranspose` here (and for `RepConv` below), contradicting the configured model default. Update all three defaults in `parse_model()`, or have these `act=True` paths resolve the configured `Conv.default_act`.

File: ultralytics/nn/modules/conv.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ class Conv(nn.Module):
...
```
Comment (UltralyticsAssistant) [inline ultralytics/nn/modules/conv.py:49]: 📝 **LOW**: The `default_act` attribute documentation for `Conv`, `ConvTranspose`, and `RepConv` still states that the default is SiLU. These class docstrings are now inaccurate and will mislead API/reference documentation users; update all three to describe LeakyReLU.

**Review by UltralyticsAssistant — COMMENTED** [review]
## 🔍 PR Review

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

The activation change is small and consistently applied to the three class defaults. However, explicitly configured YAML activation settings do not propagate to the newly independent ConvTranspose and RepConv defaults, and the public class documentation remains stale.

💬 Posted 2 inline comments

**Comment by glenn-jocher** [comment]
Thank you for taking the time to contribute and explore this change! We appreciate the effort, but changing the default Conv activation to LeakyReLU is not aligned with our current roadmap, so we'll close this PR.
